docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3.8"
+
+services:
+  mysqldb:
+    image: mysql:5.7
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_DATABASE=$MYSQLDB_DATABASE
+      - MYSQL_ROOT_PASSWORD=$MYSQLDB_ROOT_PASSWORD
+    ports:
+      - $MYSQL_LOCAL_PORT:$MYSQL_DB_DOCKER_PORT
+  app:
+    depends_on:
+      - mysqldb
+    image: atiukhtii/book-wonders
```

You may add restart to solve your problem with start-up before DB is ready to accept connection during compose startup
```suggestion
    image: atiukhtii/book-wonders
    restart: unless-stopped
```